packages/connectors/src/coinbaseWallet.ts

```diff
@@ -12,43 +10,14 @@ import {
   numberToHex,
 } from 'viem'
 
-// TODO(@3): Set `enableMobileWalletLink` to `true`
-export type CoinbaseWalletParameters = Evaluate<
-  Mutable<
-    Omit<
-      ConstructorParameters<typeof CoinbaseWalletSDK>[0],
-      'reloadOnDisconnect' // remove property since TSDoc says default is `true`
-    >
-  > & {
-    /**
-     * Fallback Ethereum JSON RPC URL
-     * @default ""
-     */
-    jsonRpcUrl?: string | undefined
-    /**
-     * Fallback Ethereum Chain ID
-     * @default 1
-     */
-    chainId?: number | undefined
-    /**
-     * Whether or not to reload dapp automatically after disconnect.
-     * @default false
-     */
-    reloadOnDisconnect?: boolean | undefined
-  }
->
-
 coinbaseWallet.type = 'coinbaseWallet' as const
-export function coinbaseWallet(parameters: CoinbaseWalletParameters) {
-  const reloadOnDisconnect = false
-
-  type Provider = CoinbaseWalletProvider
+export function coinbaseWallet(parameters: CoinbaseWalletSDKOptions) {
```

Comment:
Let's keep `CoinbaseWalletParameters` around, even if it just proxies `CoinbaseWalletSDKOptions`. Gives us more control over the connector's public API.